ui/webui/src/components/HeaderKebab.jsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with This program; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React, { useState } from "react";
+import { Dropdown, DropdownItem, DropdownPosition, KebabToggle } from "@patternfly/react-core";
+import { AnacondaAboutModal } from "./AnacondaAboutModal";
+
+export const HeaderKebab = () => {
+    const [isOpen, setIsOpen] = useState(false);
+    const [isAboutModalOpen, setIsAboutModalOpen] = useState(false);
+
+    const onToggle = isOpen => {
+        setIsOpen(isOpen);
+    };
+    const onFocus = () => {
+        const element = document.getElementById("toggle-kebab");
+        element.focus();
+    };
+    const onSelect = () => {
+        setIsOpen(false);
+        onFocus();
+    };
+
+    const handleAboutModal = () => {
+        setIsAboutModalOpen(true);
+    };
+
+    const dropdownItems = [
+        <DropdownItem key="separated link" onClick={handleAboutModal}>
+            About
+        </DropdownItem>,
+    ];
+    return (
+        <>
```

Comment:
This <> is useless, can be removed.

Reply:
that one is needed, if not i get an  `Parsing error: Unexpected token {` because of the following 
```
{isAboutModalOpen &&
            <AnacondaAboutModal
              isModalOpen={isAboutModalOpen}
              setIsAboutModalOpen={setIsAboutModalOpen}
            />}
```